akvo/rsr/spa/app/modules/reports/reports.jsx

```diff
@@ -15,6 +15,8 @@ const Reports = ({programId, userRdr}) => {
     }
   }, [userRdr])
   const orgs = userRdr && userRdr.organisations ? userRdr.organisations : []
+  const employment = currentOrg && userRdr.approvedEmployments.find(it => it.organisation === currentOrg)
```

Comment:
Is there some way we can move this to the backend? Just wondering. Ideally, I'd want to keep the permissions and roles checks on the backend, to keep them all in one place.